.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -23,3 +23,10 @@ labels: feature
 
 ### Design Requirements
 <!-- If the customization includes input from our design team, the detailed requirements will be collected here. Note: These will exist mainly in German to simplify internal communication. -->
+
+#### Link to Figma
+<!-- The UI/UX team will post the link to figma here -->
+
+#### User Story
+<!-- A clear description of the User Story that should be achived by the new feature. The User Story should be in the following pattern: 
```

Comment:
```suggestion
<!-- A clear description of the User Stories that should be achieved by the new feature. The User Stories should follow this pattern: 
```
Most of our issues and PRs are consisting of multiple User Stories. That should be reflected in the template IMO